test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionDescriptorProviderTests.cs

```diff
@@ -38,6 +38,20 @@ public void GetDescriptors_GetsDescriptorsOnlyForValidActions()
         }
 
         [Fact]
+        public void GetDescriptors_DisplayNameIncludesAssemblyName()
+        {
+            // Arrange
+            var provider = GetProvider(typeof(PersonController).GetTypeInfo());
+
+            // Act
+            var descriptors = provider.GetDescriptors();
+            var descriptor = descriptors.Single(ad => ad.Name == nameof(PersonController.GetPerson));
+
+            // Assert
+            Assert.Equal($"Microsoft.AspNetCore.Mvc.Internal.ControllerActionDescriptorProviderTests+PersonController.GetPerson (Microsoft.AspNetCore.Mvc.Core.Test)", descriptor.DisplayName);
```

Comment:
Could we do `{typeof(PersonControllerType).FullName}.{nameof(PersonController.GetPerson)}` here?